### PR TITLE
Add collections topic

### PR DESCRIPTION
**Pull request by DennisDawson**
Short topic on using the NFTokenTaxon field to label a collection of NFTs.

### CODE REVIEW

File: content/concepts/tokens/nft-collections.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+blurb: You can mint NFTs as collections using the NFT Taxon field.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Minting NFTs into Collections
+
+You can use the `NFTokenTaxon` field to group NFTs into collections. As the minter, you can choose any numeric value from `0x0` to `0xFFFFFFFF`and assign it to NFTs as you create them. The significance of the taxon is entirely up to you.
+
+For example, for your first collection, you might set the `NFTokenTaxon` to `1`. You might have a collection of NFTs with taxon values of `316`, `420`, or `911`. You might use taxons that start with a digit to indicate the type of NFT (for example, all Real Estate NFTs have a taxon that starts with `2`).
```
Comment (oeggert) [inline content/concepts/tokens/nft-collections.md:13]:
```suggestion
For example, you might set the `NFTokenTaxon` to `1`. You might have a collection of NFTs with taxon values of `316`, `420`, or `911`. You might use taxons that start with a digit to indicate the type of NFT (for example, all Real Estate NFTs have a taxon that starts with `2`).
```

File: content/references/protocol-reference/data-types/nftoken.md [hunk 2]
```diff
@@ -98,14 +98,15 @@ The fourth section is a `NFTokenTaxon` created by the issuer.
...
+Notice that the scrambled version of the `NFTokenTaxon` is `0xBC8B858E`, the scrambled version of the `NFTokenTaxon` specified by the issuer. But the _actual_ value of the `NFTokenTaxon` is the unscrambled value.
 
 ### Token Sequence
 
-The fifth section is a sequence number that increases with each `NFToken` the issuer creates. The [NFTokenMint transaction][] sets this part of the `NFTokenID` automatically based on the `MintedTokens` field of the `Issuer` account. If the issuer's [AccountRoot object][] does not have a `MintedTokens` field, the field is assumed to have the value 0; the value of the field is then incremented by exactly 1.
+The fifth section is a sequence number that increases with each `NFToken` the issuer creates.
+
+![Sequence Number](img/nftokene.png "Sequence Number")
 
+The [NFTokenMint transaction][] sets this part of the `NFTokenID` automatically based on the `MintedTokens` field of the `Issuer` account. If the issuer's [AccountRoot object][] does not have a `MintedTokens` field, the field is assumed to have the value 0; the value of the field is then incremented by exactly 1.
```
Comment (oeggert) [inline content/references/protocol-reference/data-types/nftoken.md]:
```suggestion
The [`NFTokenMint` transaction][] sets this part of the `NFTokenID` automatically based on the `MintedTokens` field of the `Issuer` account. If the issuer's [`AccountRoot` object][] does not have a `MintedTokens` field, the field is assumed to have the value 0; the value of the field is then incremented by exactly 1.
```

**Review by oeggert — APPROVED** [review]
Minor suggestions. Looks good otherwise.

**Review by tequdev — COMMENTED** [review]
It would be nice if you could write a TODO comment indicating that the Japanese translation needs to be updated.

https://github.com/XRPLF/xrpl-dev-portal/blob/6f261205d8ce170af47221e31512479b3d0dc84a/dactyl-config.yml#L2246

File: content/concepts/tokens/nft-collections.md (hunk 1 above)
Comment (DennisDawson) [reply content/concepts/tokens/nft-collections.md:13]: The point I'm making here is that you could label your first collection _1_, as in "this is the first collection." The other examples suggest other ways to make the numeric value meaningful.

**Pull request merged**